internal/nutanix/hosts.go

```diff
@@ -244,6 +244,7 @@ func NewHostsCollector(_api *Nutanix) *HostsExporter {
 				"hypervisor_memory_usage_ppm":          true,
 				"hypervisor_num_received_bytes":        true,
 				"hypervisor_num_transmitted_bytes":     true,
+				"hypervisor_full_name":                 true,
```

Comment:
why is this added here? it is a property and not part of the stats object right?